src/graph.ts

```diff
@@ -204,6 +268,75 @@ export class IndustryNode extends ConsumerNode {
     }
 }
 
+export class TransferNode extends IndustryNode {
```

Comment:
`TransferNode` units shouldn't inherit from `IndustryNode` as transfer units can transfer all items, not just craftables.

Reply:
You're right! Fixed in https://github.com/lgfrbcsgo/du-factory-generator/commit/b68fb5911d51113ba7f1700a7a570d463a554085